src/main/java/ui/battle/BattleView.java

```diff
@@ -249,8 +263,13 @@ public BattleView(IView parent) {
 
                         Label battleInfo = new Label();
                         battleInfo.setPadding(new Insets(30, 0, 5, 0));
-                        battleInfo.textProperty().bind(
-                                source.nameProperty().concat(" greift ").concat(target.nameProperty()).concat(" an."));
+                        battleInfo.textProperty().bind(source.nameProperty()
+                                .concat(" ")
+                                .concat(LanguageUtility.getMessageProperty("battle.info.attacks.verb"))
+                                .concat(" ")
+                                .concat(target.nameProperty())
+                                .concat(" ")
```

Comment:
only append this space if battle.info.attacks.ending is not empty
(SomeOne attacks AnotherOne)
vs
(SomeOne greif AnotherOne an)

alternatively, trim the result